### DISCUSSION TITLE
discord server

### DISCUSSION THREAD

**Discussion by nevarim** [jrmi/airboardgame#463] [General]
hi
is there a discord server for this program?
Nevarim

**Comment by jrmi** [comment]
Sure, I should improve the visibility but look at the repository description page:

![image](https://github.com/jrmi/airboardgame/assets/571533/cb58d17d-59eb-41da-a4c0-f8dd8e8bfff8)

The link is https://discord.gg/EsZGJ5h6UA

Thank you for your interest :)
